packages/cubejs-schema-compiler/src/adapter/BigqueryQuery.ts

```diff
@@ -83,7 +83,7 @@ export class BigqueryQuery extends BaseQuery {
    * @see https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#interval_type
    * It returns a tuple of (formatted interval, timeUnit to use in datediff functions)
    */
-  private formatInterval(interval: string): [string, string] {
+  public formatInterval(interval: string): [string, string] {
```

Comment:
There is no `public formatInterval()` in BaseQuery, thus this function is not available in all dialects. So it is not safe to use it publicly...